### DISCUSSION TITLE
How to remove guidelines?

### DISCUSSION THREAD

**Discussion by lcrodrigues** [patrykandpatrick/vico#764] [Questions]
### Question

I'm trying to make the second chart, and the first chart is the closest that I got. Is it possible to remove some guidelines like that first vertical line, the bottom axis line and the ticks indicating the bottom axis values?

![image 10](https://github.com/patrykandpatrick/vico/assets/16669026/78b57ac1-e6a6-471b-a776-691382382932)

![Frame 1000003882](https://github.com/patrykandpatrick/vico/assets/16669026/76c19bb8-fd0f-4b00-bc86-6ed5615f6c5b)

### Vico version(s)

1.14.0

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello! Yes, just set `axis` and `tick` to `null` in [`rememberBottomAxis`](
https://patrykandpatrick.com/vico/api/1.14.0/compose/com.patrykandpatrick.vico.compose.axis.horizontal/remember-bottom-axis) and [`rememberStartAxis`](https://patrykandpatrick.com/vico/api/1.14.0/compose/com.patrykandpatrick.vico.compose.axis.vertical/remember-start-axis).
